python/lsst/jointcal/dataIds.py

```diff
@@ -59,22 +59,19 @@ def makeDataRefList(self, namespace):
         if self.datasetType is None:
             raise RuntimeError("Must call setDatasetType first")
         skymap = None
-        log = None
+        log = lsst.log.Log.getLogger("jointcal.dataIds")
```

Comment:
Using namespace.log gives the log output as `root`, which doesn't seem as helpful when figuring out where the message came from. If that's the standard system though, I'll use it.